iso15118/secc/states/iso15118_2_states.py

```diff
@@ -586,6 +586,11 @@ async def process_message(
             msg.body.certificate_installation_req
         )
 
+        # For the CertificateInstallation, the min the EVSE can do is
+        # to verify the message signature, using the OEM provisioning
+        # certificate (public key)
+        # The chain of certificates of the OEM Root can also be verified,
```

Comment:
Slight clarification: you don't verify certificates, you verify signatures. I'd write:
"The chain of signatures, from the signature in the CertificateInstallationReq's header all the way to the self-signed OEM/V2G root certificate, can be verified if the OEM Sub-CA and OEM/V2G root CA certificate is available."